resources/postgresql_servers.go

```diff
@@ -203,64 +204,16 @@ func PostgresqlServers() *schema.Table {
 				Name:        "azure_postgresql_server_private_endpoint_connections",
 				Description: "Azure postgresql server private endpoint connection",
 				Resolver:    fetchPostgresqlServerPrivateEndpointConnections,
+				Options:     schema.TableCreationOptions{PrimaryKeys: []string{"server_cq_id", "id"}},
 				Columns: []schema.Column{
 					{
-						Name:        "server_id",
+						Name:        "server_cq_id",
 						Description: "Unique ID of azure_postgresql_servers table (FK)",
 						Type:        schema.TypeUUID,
 						Resolver:    schema.ParentIdResolver,
 					},
 					{
-						Name:        "resource_id",
```

Comment:
Looks like a lot of fields were deleted. They were not: there was a duplicate relation that I have dropped, so it looks a bit not apparent in diff here.